package.json

```diff
@@ -18,11 +18,28 @@
     "type": "git",
     "url": "https://github.com/kaimallea/isMobile.git"
   },
-  "main": "./isMobile",
+  "scripts": {
+    "test": "jest --coverage",
+    "dev": "jest --watch",
+    "prepublish": "rimraf dist && npm run build && npm run minify",
```

Comment:
why do we need an npm package to delete a directory?

Reply:
🤷‍♂️. We don't actually